app/src/main/java/com/example/memeapp/data/mapper/MemeRealmMapper.kt

```diff
@@ -0,0 +1,21 @@
+package com.example.memeapp.data.mapper
+
+import com.example.memeapp.data.local.model.MemeRealmEntity
+import com.example.memeapp.domain.entity.MemesEntity
+
+object MemeRealmMapper {
```

Comment:
this is not fine as the extensions not need an object. Please remove the object, rename the file as MapperExtensions.kt and put the extensions on it

Reply:
I think that another way is to put the extensions bellow de MemesEntity. It more close to use when you see the class file

Reply:
Done thx